konfera/event/views.py

```diff
@@ -127,3 +128,16 @@ def event_public_tickets(request, slug):
 
     context['tickets'] = available_tickets
     return render(request=request, template_name='konfera/event_public_tickets.html', context=context)
+
+
+def event_order_detail(request, slug, order_uuid):
+    context = dict()
+    event = get_object_or_404(Event.objects.published(), slug=slug)
+    context['event'] = event
+    order = get_object_or_404(Order, uuid=order_uuid)
+    context['order'] = order
+
+    if order:
```

Comment:
This condition is unnecessary. `order` cannot be anything else than an object because a 404 would be raised.